public/app/plugins/datasource/prometheus/mode-prometheus.js

```diff
@@ -75,8 +75,7 @@ var PrometheusHighlightRules = function() {
       regex : '"[^"]*"|\'[^\']*\''
     }, {
       token : "punctuation.operator",
-      regex : ",",
-      push  : 'start-label-matcher'
+      regex : ","
```

Comment:
I'm not understand well, but the label match auto complete work without pushing `start-label-matcher`.

Reply:
Yes, the `start-label-matcher` is referenced at https://github.com/grafana/grafana/pull/9555/files#diff-52972fb05f645321f656a0cc5f9c99d4R56 .

Reply:
And the role of `keyword.operator` is not same.
One is for calculation, another one is for label match.